qiskit/ignis/verification/tomography/fitters/gateset_fitter.py

```diff
@@ -56,16 +56,16 @@ def __init__(self,
             and the gateset basis used when collecting the data.
 
         Example:
-            >> gate = HGate()
-            >> basis = default_gateset_basis()
-            >> basis.add_gate(gate)
-            >> backend = ...
-            >> circuits = gateset_tomography_circuits(gateset_basis=basis)
-            >> qobj = assemble(circuits, shots=10000)
-            >> result = backend.run(qobj).result()
-            >> fitter = GatesetTomographyFitter(result, circuits, basis)
-            >> result_gates = fitter.fit()
-            >> result_gate = result_gates[gate.name]
+            >>> gate = HGate()
+            >>> basis = default_gateset_basis()
+            >>> basis.add_gate(gate)
+            >>> backend = ...
+            >>> circuits = gateset_tomography_circuits(gateset_basis=basis)
+            >>> qobj = assemble(circuits, shots=10000)
+            >>> result = backend.run(qobj).result()
+            >>> fitter = GatesetTomographyFitter(result, circuits, basis)
+            >>> result_gates = fitter.fit()
+            >>> result_gate = result_gates[gate.name]
```

Comment:
Instead of using `>>>` like this is a python interpreter you should use a code block here, especially to get syntax highlighting. Something like:

```rst
Example::

    from qiskit.circuits.library.standard import *
    from qiskit.ignis.verification.basis import default_gateset_basis           
    from qiskit.ignis.verification import gateset_tomography_circuits
    from qiskit.ignis.verification import GateSetTomographyFitter

    gate = HGate()
    basis = default_gateset_basis()
    basis.add_gate(gate)
    backend = ...
    circuits = gateset_tomography_circuits(gateset_basis=basis)
    qobj = assemble(circuits, shots=10000)
    result = backend.run(qobj).result()
    fitter = GatesetTomographyFitter(result, circuits, basis)
    result_gates = fitter.fit()
    result_gate = result_gates[gate.name]
```

Reply:
Done, thanks.